src/test/scala/com/gu/subscriptions/cas/directives/ProxyDirectiveSpec.scala

```diff
@@ -1,5 +1,6 @@
 package com.gu.subscriptions.cas.directives
```

Comment:
We could test the zuoraRoute: 

- [ ] Check that we return a `notFound` if the subscription doesn't exist
- [ ] Check that we set the `activationDate`